pgsql_big_dedupe_example/pgsql_big_dedupe_example.py

```diff
@@ -195,9 +188,9 @@
 print 'creating inverted index'
 
 for field in deduper.blocker.index_fields:
-    c2 = con2.cursor('c2')
+    c2 = con.cursor('c2')
     c2.execute("SELECT DISTINCT %s FROM processed_donors" % field)
-    field_data = (row for row in c2)
+    field_data = set(row[field] for row in c2)
```

Comment:
It would be better to have this just be 

`(row[field] for row in c2)`

that way the whole object doesn't have to read into memory.  The distinct sql statement should make sure it's unqiue